Instruments/config/2017/Skimmer.cfg

```diff
@@ -4,7 +4,9 @@ mode: HH
 jet_ordering: DeepFlavour
 #jet_ordering: HHJetTag
 btag_wp: Medium
-#cachePath: cache_tuple
+cachePath: eTau
+#cachePath: muTau
+#cachePath: tauTau
```

Comment:
- cache paths should be None. Channel name should be added in the skimmer code.
- add cache paths for 2016 and 2018 as well.